logic/in_transfer.js

```diff
@@ -92,8 +92,7 @@ InTransfer.prototype.verify = function(transaction, sender, cb, tx) {
 	}
 
 	const amount = new Bignum(transaction.amount);
-
-	if (!amount.equals(0)) {
+	if (amount.lessThanOrEqualTo(0)) {
```

Comment:
We should only check for `0`, as negative value is checked here:
https://github.com/LiskHQ/lisk/blob/4ae8553042b93d3e7ce41c11a7d910fef24ba644/logic/transaction.js#L643-L649

Reply:
Its consistent  across other transaction types. 

https://github.com/LiskHQ/lisk/blob/4ae8553042b93d3e7ce41c11a7d910fef24ba644/logic/transfer.js#L85-L87

Reply:
Do you think we should follow this "consistency"?

Reply:
@4miners Yes at least for `1.1.0`. We try to avoid more code change for this release. We can fix this logic across all transactions separately in next release.

Reply:
Ok, I agree, but can you open an issue for this then?

Reply:
@nazarhussain what you eventually mean to say is that we will create a separate issue and fix the `consistency` in next release? if that is the case I have one concern, I remember Usman mentioning about not creating issues which are not worthy to be an issue to tackle separately, so if you guys agree to create a separate issue I am fine with it.

Reply:
I am also advocate of this approach to fix small stuff in between issues, but I was just worried about the time of release. But as discussed we will do a complete QA cycle afterwards, so I will make the relevant changes.